apps/snitch_core/test/support/factory/zone.ex

```diff
@@ -12,6 +12,20 @@ defmodule Snitch.Factory.Zone do
         }
       end
 
+      def state_zone_member_factory do
+        %StateZoneMember{
+          zone: build(:zone, zone_type: "S"),
```

Comment:
@arjun289 @jyotigautam we'll have to change this to use enums. @jyotigautam can you please create a story for it in pivotal tracker.